docs/templates/eia861_child.rst.jinja

```diff
@@ -18,12 +28,36 @@ details, see the `official EIA-861 page <{{ source.path }}>`__
 {% endblock %}
 
 {% block availability %}
+PUDL incorporates all available EIA-861 data (except the 2019 data) dating back to 2001.
+Earlier data from the forms EIA-861 (utility, Sales to Ultimate Customers, and Operational data),
+and EIA-861S (Sales to Ultimate Customers) exists for the years 1990-2000, but is not yet included in PUDL.
```

Comment:
Also, the short form was created in 2012 so that bit is unnecessary. I would just say: 

"Earlier data (1990-2000) exists but has not been incorporated into PUDL yet.